noxfile.py

```diff
@@ -1,14 +1,24 @@
-# Copyright (c) 2017 pandas-gbq Authors All rights reserved.
-# Use of this source code is governed by a BSD-style
-# license that can be found in the LICENSE file.
-
-"""Nox test automation configuration.
-
-See: https://nox.readthedocs.io/en/latest/
-"""
-
+# -*- coding: utf-8 -*-
```

Comment:
This file is templated: https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/noxfile.py.j2